tensorflow/core/kernels/conv_grad_filter_ops.cc

```diff
@@ -112,6 +113,22 @@ struct LaunchConv2DBackpropFilterOp<CPUDevice, T> {
   }
 };
 
+#ifdef TF_USE_SYCLEIGEN
+template <typename T>
+struct LaunchConv2DBackpropFilterOp<SYCLDevice, T> {
+  void operator()(OpKernelContext* ctx, bool use_cudnn, bool cudnn_use_autotune,
+                  const Tensor& out_backprop, const Tensor& input,
+                  int row_stride, int col_stride, const Padding& padding,
+                  Tensor* filter_backprop, TensorFormat data_format) {
+    const SYCLDevice& d = ctx->eigen_device<SYCLDevice>();
+    functor::SpatialConvolutionBackwardKernel<SYCLDevice, T>()(
+        d, filter_backprop->tensor<T, 4>(), input.tensor<T, 4>(),
+        out_backprop.tensor<T, 4>(), filter_backprop->dim_size(0),
+        filter_backprop->dim_size(1), row_stride, col_stride);
+  }
+};
+#endif  // TENSORFLOW_USE_SYCL
```

Comment:
// TF_USE_SYCLDNN
instead of 
// TENSORFLOW_USE_SYCL

Reply:
Oh yeah actually should be TF_USE_SYCLEIGEN. Noticed 2 other places like that